src/translator/sentence_ranges.h

```diff
@@ -1,52 +1,253 @@
+/// It's best not to look at the horrors in this file at this stage.
```

Comment:
It sounds like you don't want me to review this...

Reply:
`absl::string_view`, `std::string_view` everything has been ridden off to get the horror out. 


`Annotation` composed off `ByteRange` is independent of `string_view`. 

An `AnnotatedBlob = (std::string, Annotation)` now manages the conversion between our `absl::string_view` and the internal `ByteRange` (since it has access to the std::string string_views refer to) fixing the remaining code outside which used to operate with this class without major invasive changes.